app/build.gradle

```diff
@@ -75,6 +75,8 @@ dependencies {
 
     implementation "com.google.android.material:material:1.4.0"
 
+    implementation 'com.beust:klaxon:5.5'
```

Comment:
Im just curious why do we need this klaxon library?

Reply:
To work with json, do you mean replace it with kotlinx.serialization?